rdr_service/dao/genomics_dao.py

```diff
@@ -1966,6 +2026,27 @@ def get_raw_record_from_bid_genome_type(self, *, biobank_id, genome_type):
 
             return record
 
+    def get_set_member_deltas(self):
+        with self.session() as session:
+            return session.query(
+                GenomicAW1Raw
+            ).outerjoin(
+                GenomicSetMember,
+                GenomicSetMember.sampleId == GenomicAW1Raw.sample_id
+            ).filter(
+                GenomicSetMember.id.is_(None),
+                GenomicAW1Raw.ignore_flag == 0,
+                GenomicAW1Raw.biobank_id != "",
+                GenomicAW1Raw.sample_id != "",
+                GenomicAW1Raw.collection_tube_id != "",
+                GenomicAW1Raw.test_name != "",
```

Comment:
Should we lean into alchemy more and use .`isnot(None)` here?

Reply:
I'm specifically filtering for blank strings here.